SendITapp/api/v2/views/userView.py

```diff
@@ -118,16 +118,16 @@ def post(self):
         if data["email"].find("@") < 2:
             message = 'Incorrect email format'
             payload = {"Status": "Failed", "Message": message}
-            answ = make_response(jsonify(payload), 400)
-            return answ
+            return make_response(jsonify(payload), 400)
+            
 
     
         password = data['password'].strip()
         if not (re.match("^[a-zA-Z0-9_]*$", password)):
             message = 'Check your password.'
             payload = {"Status": "Failed", "Message": message}
-            answ = make_response(jsonify(payload), 400)
-            return answ
+            return make_response(jsonify(payload), 400)
+            
         
         user = UserModel()
```

Comment:
For `UserModel.login()`, validate the password he/she provides is equal to the one in the DB.